tests/test_code_location_changes.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=duplicate-code
+"""Tests for code location changes checker."""
+import astroid
+from pylint.testutils import CheckerTestCase
+from pylint.testutils import MessageTest
+from pytest import mark
+
+from pylint_nautobot.code_location_changes import NautobotCodeLocationChangesChecker
+
+
+class TestCodeLocationChangesChecker(CheckerTestCase):
+    """Test model label construction checker."""
+
+    CHECKER_CLASS = NautobotCodeLocationChangesChecker
+
+    @mark.parametrize(
+        ("test_code", "expected_args"),
+        (
+            # TODO: Fix the following test param, should emit a message too
```

Comment:
Should I fix the parser? @Kircheneer

Reply:
This is because I guess the whole module `nautobot.core.fields` doesn't exist anymore? If its not too invasive of a change, fine for me.